### PR TITLE
add a full list of transitions to readme

### PR DESCRIPTION
**Pull request by tristansokol**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -695,7 +695,7 @@ You can also use different in and out transitions for the same slide:
     And it starts again.
 </section>
 ```
-
+You can choose from `none` `fade` `slide` `convex` `concave` `zoom`.
```
Comment (mschilli87) [inline README.md]: "``You can choose from `none`, `fade`, `slide`, `convex`, `concave`, and `zoom`.``"?

Comment (tristansokol) [reply README.md]: Sorry, I forgot to fill in the description. I couldn't find a list available transitions, so I was hoping to get them enumerated in the docs somewhere.

Comment (mschilli87) [reply README.md]: Sorry I was not clear in my comment: I was trying to suggest adding commas and a final conjunction ("and") to the new sentence. :wink:

**Comment by tristansokol** [comment]
@mschilli87 added some grammar!

**Comment by hakimel** [comment]
🙌

**Pull request merged**